platinum-push-messaging.html

```diff
@@ -303,7 +303,7 @@
         }).map(function(propName) {
           return JSON.stringify(propName) + ':' + JSON.stringify(options[propName]);
         });
-        return '{' + parts.join(',') + '}';
+        return encodeURIComponent('{' + parts.join(',') + '}');
```

Comment:
The decode is in the service worker, https://github.com/PolymerElements/platinum-push-messaging/blob/master/service-worker.js#L11

This fix is actually to add the missing encode to match the existing decode.